00_Basic_DifferentFolder/mock/mockup.gv

```diff
@@ -0,0 +1,32 @@
+digraph Basic_DifferentFolder {
+    
+    
+	node [style=bold, shape=box];
+	a[label="abund.csv"];
+	b[label="CSVNormalize", color=deepskyblue, style=filled, fontcolor=white];
+	c[label="abund.norm.csv"];
+	d[label="Spearman", color=deepskyblue, style=filled, fontcolor=white];
+	e[label="spearmancsv"];
+	f[label="CSVPad", color=deepskyblue, style=filled, fontcolor=white];
+	g[label="Networks/network.csv"];
+	h[label="GPTAtria", color=deepskyblue, style=filled, fontcolor=white];
+	i[label="CSV2GML", color=deepskyblue, style=filled, fontcolor=white];
+	j[label="ATria.noa"];
+	k[label="Networks/network.gml"];
+
+	
+a -> b;
```

Comment:
Could we try fixing this formatting? And ensure that all edges are tabbed? Also we are no longer going to label vertices with letters. Take a look at: https://github.com/movingpictures83/PipeViz/blob/master/00_Basic_DifferentFolder/expected.dot